src/components/CorporateCampaign/LoanSearch/LoanSearchFilters.vue

```diff
@@ -352,7 +352,6 @@ export default {
 				return !this.excludedTags.includes(tag.name);
 			});
 		});
-		this.setLoanDisplayMode(this.activeLoanDisplay);
```

Comment:
This was triggering the `showLoans` mechanisms immediately upon page load. It the flows into the loan display and fired a default loan query before we knew anything about the promotion resulting in potentially ineligible loans to show on the page.

Reply:
I've verified that the default `rows` view and the contentful setting that swaps to `grid` both work and load up properly.